spec/localeapp/exception_handler_spec.rb

```diff
@@ -26,10 +26,15 @@
   end
 
   it "handles when the default is a Symbol that can't be resolved" do
-    expect(Localeapp.missing_translations).to receive(:add).with(:en, :foo, nil, {:default => :bar})
+    expect(Localeapp.missing_translations).to receive(:add).with(:en, 'foo', nil, {:default => :bar})
     I18n.t(:foo, :default => :bar)
   end
 
+  it "escapes html tags from keys to prevent xss attacks" do
+    expect(Localeapp.missing_translations).to receive(:add).with(:en, '&lt;script&gt;alert(1);&lt;/script&gt;', nil, {})
+    expect(I18n.t('<script>alert(1);</script>')).to eq '&lt;script&gt;alert(1);&lt;/script&gt;'
```

Comment:
The travis failures are genuine here: the locale is missing at the start of the string. It should be:
`expect(I18n.t('<script>alert(1);</script>')).to eq 'en.&lt;script&gt;alert(1);&lt;/script&gt;'`